x-pack/plugins/fleet/server/services/epm/kibana/index_pattern/install.ts

```diff
@@ -11,6 +11,7 @@ import { dataTypes, installationStatuses } from '../../../../../common/constants
 import { ArchivePackage, Installation, InstallSource, ValueOf } from '../../../../../common/types';
 import { RegistryPackage, DataType } from '../../../../types';
 import { getInstallation, getPackageFromSource, getPackageSavedObjects } from '../../packages/get';
+import { IndexPatternAttributes } from '../../../../../../../../src/plugins/data/common/index_patterns';
```

Comment:
nit: could you please fix import. '../../../../../../../../src/plugins/data/server'